android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/AccountDAO.java

```diff
@@ -0,0 +1,29 @@
+package org.sagebionetworks.bridge.android.manager;
+
+import android.support.annotation.AnyThread;
+import android.support.annotation.Nullable;
+
+import org.sagebionetworks.bridge.rest.model.SignIn;
+import org.sagebionetworks.bridge.rest.model.StudyParticipant;
+import org.sagebionetworks.bridge.rest.model.UserSessionInfo;
+
+/**
+ * Created by jyliu on 2/8/2017.
+ */
+@AnyThread
+public interface AccountDAO {
```

Comment:
What's the benefit of abstracting this class as an interface?  Seems confusing to me.  How about just making it a class and deleting InMemoryDao?